AssimpNet/Unmanaged/AssimpLibrary.cs

```diff
@@ -1464,6 +1464,9 @@ internal static class AssimpDefaultLibraryPath
         public const String DefaultWindows32BitPath = "Assimp32.dll";
         public const String DefaultWindows64BitPath = "Assimp64.dll";
 
+        public const String DefaultUWP32BitPath = "assimp-vc140-mt.dll";

```

Comment:
What's up with these names? Are they not configurable with this vcpkg thing? It's not a problem one way or the other, they just seem odd.